ikuzo/storage/memory/organization_test.go

```diff
@@ -37,13 +37,32 @@ func TestMemoryStore(t *testing.T) {
 	orgID, err := domain.NewOrganizationID("demo")
 	is.NoErr(err)
 
-	err = store.Put(ctx, domain.Organization{ID: orgID})
+	org := &domain.Organization{
+		ID: orgID,
+		Config: domain.OrganizationConfig{
+			Domains: []string{"localhost", "demo.localhost"},
+		},
+	}
+
+	err = store.Put(ctx, org)
 	is.NoErr(err)
+	is.Equal(len(store.organizations), 1)
+	is.Equal(len(store.domains), len(org.Config.Domains))
 
 	// should have one org
 	orgs, err = store.Filter(ctx)
 	is.NoErr(err)
-	is.Equal(len(orgs), 1)
+	is.Equal(len(orgs), len(store.organizations))
+
+	// filter unknown domain
```

Comment:
Do we need to keep this?

Reply:
For know.  In a later pull we will add more integration tests for this.